_posts/en/2016-12-28-delegates-closures.md

```diff
@@ -0,0 +1,299 @@
+---
+layout: post
+title: Delegates vs Closures
+lang: en
+permalink: /delegates-closures/
+excerpt: "Delegates VS Closures"
+authors:
+    - thuchon
+date: '2016-12-28 18:25:24 +0100'
+date_gmt: '2016-12-28 17:25:24 +0100'
+categories:
+- Mobile
+tags:
+- mobile
+- ios
+- delegates
+- block
+- closure
+---
+
+### Introduction
+
+Hi Astronauts, today we are going to talk about mobile programming, et you know what it's native development !!! This article will be for beginners, so we will spend a bit of time to understand few technical terms together before going deep in the subject. This time we will focus on iOS, but don't worry, I plan to do the equivalent for Android as soon as possible.
```

Comment:
and you know what, it's native development!!!
to understand a few technical terms